docs/guidance/EnterpriseScaleAnalytics-DatabricksManualSetup.md

```diff
@@ -8,9 +8,9 @@ Due to the following service limitations, Databricks needs to be setup manually:
 
 ## Manual Databricks configuration
 
-Due to the issue mentioned above, we cannot rely on the application workflow, but only rely on the on-behalf workflow. This means, that instead of using a Service Principle for authentication, we need to rely on a user being authenticated against the workspace. Only then, we can use the Databricks API to create a Key Vault backed secret scopes. If you are ok with Databricks backed secret scopes, then you can already automate the complete setup end-to-end. However, for manageability reasons, we are recommending to use Key Vaults for storing secrets.
+Due to the issue mentioned above, we cannot rely on the application workflow, but only rely on the on-behalf workflow. This means, that instead of using a Service Principle for authentication, we need to rely on a user being authenticated against the workspace. Only then, we can use the Databricks API to create a Key Vault backed secret scopes. If you are OK with Databricks backed secret scopes, then you can already automate the complete setup end-to-end. However, for manageability reasons, we are recommending to use Key Vaults for storing secrets.
```

Comment:
Typo - change "principle" to "principal"

"we are recommending to use Key Vaults for storing secrets." change to "we recommend using Azure Key Vault for storing secrets."